.eslintrc.json

```diff
@@ -4,6 +4,7 @@
   "plugins": ["prettier"],
   "rules": {
     "import/no-extraneous-dependencies": false,
+    "import/prefer-default-export": false,
```

Comment:
👍 if you also agree, we should do that in every project actually.

Reply:
Yeah I agree :)

I suggest we create an eslint rules package like `eslint-config-airbnb` to keep these things easy and consistent

Reply:
yeah. good idea